data/redirects.json

```diff
@@ -487,6 +487,11 @@
     "from":"meetings/2015/01/06/january-2015-meeting",
     "to":"/meetings/2015/january/"
   },
+  // redirects for mistakes
```

Comment:
Apparently, JSON doesn't support comments, but as far as I can tell, the ruby parser for it allows them, so 🤷‍♂️